wasm/src/main/java/com/dylibso/chicory/wasm/ParserUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.dylibso.chicory.wasm;
+
+public class ParserUtil {
+
+    private ParserUtil() {}
+
+    public static boolean isValidIdentifier(String s) {
+        if (s == null) {
+            return false;
+        }
+        for (int i = 0; i < s.length(); i++) {
+            if (!isValidIdentifierChar(s.codePointAt(i))) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    private static boolean isValidIdentifierChar(int ch) {
+        return Character.isUnicodeIdentifierPart(ch)
+                || ch == '-'
+                || ch == '>'
+                || ch == ' '
+                || ch == '.'
+                || ch == '⌣';
```

Comment:
Haven't noticed this before, looks like a very specific character, do you have a reference to the spec or is it just empirical?

Reply:
I know it's very specific and pretty weird. If I don't do this all tests generated from `custom.wast` fail. I'm pretty sure there should a more a general way to check the validity of a character, but so for I haven't find any.

Reply:
I went back to the specification and import names are defined as [Names](https://webassembly.github.io/spec/core/syntax/values.html#names) and all the Unicode characters are allowed.
I hope we can come up with a more principled check here.

Reply:
Thanks for pointer, following it I found a more reasonable way to test allowed chars.

Reply:
Nice! Thanks! 👍